crates/katana/rpc/rpc-api/src/torii.rs

```diff
@@ -0,0 +1,11 @@
+use jsonrpsee::core::RpcResult;
+use jsonrpsee::proc_macros::rpc;
+use katana_rpc_types::transaction::{TransactionsPage, TransactionsPageCursor};
+
+#[cfg_attr(not(feature = "client"), rpc(server, namespace = "torii"))]
+#[cfg_attr(feature = "client", rpc(client, server, namespace = "torii"))]
```

Comment:
@kariy suggested in a discussion we had that we keep things under `katana` namespace.

```suggestion
#[cfg_attr(not(feature = "client"), rpc(server, namespace = "katana"))]
#[cfg_attr(feature = "client", rpc(client, server, namespace = "katana"))]
```

The idea was then to see how some endpoints can be available in `--dev` or always. As for Saya too some extra endpoints may be required and available all the time. Not only in dev.

At impl level, we may then have `KatanaDev` which is different from `Katana`. A good first issue identified by @kariy. :+1:

Reply:
I think it might've came across wrong in our other convo, but I was referring to only the endpoint for retrieving Katana configs, which I believe should be under the `katana_*` namespace, but other than that we can, if it seems appropriate, create a new namespace.

Reply:
Fwiw, I would like to have the options for user to specifically choose what endpoints to expose directly from the CLI. Kinda like how `geth` does it;

```
geth --http --http.api eth,net,web3
```` 

then for katana the values for `--http.api` could be like `starknet,katana,torii,dev` etc..

Reply:
my thinking here was a new `torii_` namespace makes sense, since other sequencers in the future might want to implement it to support torii sync. i'm happy to go with a different approach though

Reply:
> I think it might've came across wrong in our other convo

I surely get it wrong then, sorry for the clarification. :)

> I think it'd be nicer if we introduce a dev namespace for the dev oriented write endpoints

Yes definitely, this will ensure people are voluntary activating something that can have security concerns.

> Fwiw, I would like to have the options for user to specifically choose what endpoints to expose directly from the CLI.

That looks pretty nice yes, we should have a good config by default to ensure in most cases it does not need to be configured though.

> my thinking here was a new torii_ namespace makes sense, since other sequencers in the future might want to implement it to support torii sync. i'm happy to go with a different approach though

It makes totally sense to go with new namespace yes.